salt/modules/pkgin.py

```diff
@@ -181,7 +181,9 @@ def latest_version(*names, **kwargs):
 
         out = __salt__['cmd.run'](cmd, output_loglevel='trace')
         for line in out.splitlines():
-            p = line.split(',' if _supports_parsing() else None)
+            if line.startswith('No results found for'):
+                return pkglist
+            p = line.split(';' if _supports_parsing() else None)
```

Comment:
This appears to have been inadvertently changed from ';' to ',' in 9df3dab883d21d1dcf40142d233fd005fc24c9c8